Technical-Workshop/Presentation/Home Screen/ViewModel/HomeScreenViewModel.swift

```diff
@@ -8,7 +8,17 @@
 import Foundation
 
 class HomeScreenViewModel: HomeScreenViewModelType {
-    private let categoryItems = [CategoryItem(title: "Popular", image: .firstCategory), CategoryItem(title: "Breakfast", image: .secondCategory), CategoryItem(title: "Lunch", image: .thirdCategory), CategoryItem(title: "Dinner", image: .fourthCategory), CategoryItem(title: "Dessert", image: .fifthCategory)]
+    private let categoryItems = [CategoryItem(title: "Popular", image: .firstCategory),
+                                 CategoryItem(title: "Breakfast", image: .secondCategory),
+                                 CategoryItem(title: "Lunch", image: .thirdCategory),
+                                 CategoryItem(title: "Dinner", image: .fourthCategory),
+                                 CategoryItem(title: "Dessert", image: .fifthCategory)]
+    
+    private let mealItems = [MealItem(mealRecipe: "meal1", chefName: "name1", mealType: "mealtype1", servings: "servings1", image: .recipePlaceholder),
```

Comment:
some thing that came to my mind to discuss we could have used json and json reader to make a fake api call this may help reducing the code size when reusing these files in the mock api when it comes to write unit tests